File: 1544-make-the-string-great/1544-make-the-string-great.cpp
class Solution {
public:
    string makeGood(string s) {
        string ans = "";
        for(int i = 0; i<s.size(); i++) {
            if(ans != "" && abs(ans.back() - s[i]) == 32) {
                ans.pop_back();
            }
            else {
                ans += s[i];
            }
        }

        return ans;
    }
};

File: 1544-make-the-string-great/1544-make-the-string-great.py
class Solution(object):
    def makeGood(self, s):
        ans = []
        for char in s:
            if ans and abs(ord(ans[-1]) - ord(char)) == 32:
                ans.pop()
            else:
                ans.append(char)
        return ''.join(ans)
        